build.gradle

```diff
@@ -423,16 +423,16 @@ task stopEC2(type: AWSLambdaInvokeTask) {
 //////////////////////////////////////////////////////////////////////
 task generateConfig {
     doLast {
-        if (!project.file('build/tscfg-0.8.1.jar').exists()) {
+        if (!project.file('build/tscfg-0.9.4.jar').exists()) {
             download {
-                src 'https://github.com/carueda/tscfg/releases/download/v0.8.1/tscfg-0.8.1.jar'
+                src 'https://github.com/carueda/tscfg/releases/download/v0.9.4/tscfg-0.9.4.jar'
```

Comment:
Consider using gradle.properties. This version is repeated 5 times:

# Suggestion

gradle.properties:
```Groovy  
tscfgVersion = 0.9.4
```
build.gradle:
```Groovy
src "https://github.com/carueda/tscfg/releases/download/v0.9.4/tscfg-$tscfgVersion.jar"
```